src/fontr/fontr/transforms.py

```diff
@@ -0,0 +1,65 @@
+import torchvision.transforms as transforms
+import torch
+import random
+
+
+class Patch(torch.nn.Module):
+    """
+    Creates patches from images
+    """
+
+    def __init__(self, patch_size=10, patch_width=96, patch_height=96):
+        self.patch_size = patch_size
```

Comment:
They sample "fifteen test patches, each of which comes with different aspect ratios and views, from one test image" in the original paper. I guess the solution we have can work as a substitute though.